src/containers/Network/Validators.tsx

```diff
@@ -34,7 +46,7 @@ export const Validators = () => {
           newValidatorList[v.signing_key] = v
```

Comment:
can this code be incorporated into the other loop so we dont have to iterate twice?